tests/integration/test_subscriptions_cli.py

```diff
@@ -80,22 +80,26 @@ def test_subscriptions_create_failure(monkeypatch):
     assert _count_fake_subs() == 0
 
 
-def test_subscriptions_create_success(monkeypatch):
+# This subscription request has the members required by our fake API.
+# It must be updated when we begin to test against a more strict
+# imitation of the Planet Subscriptions API.
+GOOD_SUB_REQUEST = {'name': 'lol', 'delivery': True, 'source': 'wut'}
+
+
+@pytest.mark.parametrize('cmd_arg, runner_input',
+                         [('-', json.dumps(GOOD_SUB_REQUEST)),
+                          (json.dumps(GOOD_SUB_REQUEST), None)])
+def test_subscriptions_create_success(cmd_arg, runner_input, monkeypatch):
```

Comment:
We intend that planet-subscriptions-create can take a JSON subscription request from a file (including stdin) or string. The new tests parametrize the command's argument and stdin input to cover both these cases.